ui/src/views/image/TemplateZones.vue

```diff
@@ -66,7 +66,7 @@
           @onClick="showCopyTemplate(record)" />
         <tooltip-button
           style="margin-right: 5px"
-          :disabled="!('deleteTemplate' in $store.getters.apis)"
+          :disabled="!('deleteTemplate' in $store.getters.apis && !record.status.startsWith('Installing'))"
```

Comment:
hm, not (something and (not something else)) can this be re-written like
```suggestion
          :disabled="record.status.startsWith('Installing') || !('deleteTemplate' in $store.getters.apis)"
```

Reply:
Thanks fixed